cmd/deploy-with-helm/main.go

```diff
@@ -251,6 +251,11 @@ func main() {
 	}
 	fmt.Println(string(stdout))
 
+	// Collect values to be set via the CLI.
+	cliValues := []string{
+		fmt.Sprintf("--set=image.tag=%s", ctxt.GitCommitSHA),
```

Comment:
I like that this favor using an explicit argument as it will be easier to debug compared to values in a temporary generated file.